app/vmalert/config/parser.go

```diff
@@ -0,0 +1,12 @@
+package config
+
+// AlertRule basic alert entity rule
+type AlertRule struct{}
+
+// Alerts grouping array of alert
+type Alerts struct{}
+
+// Parse parses config from given file
+func Parse(filepath string) Alerts {
```

Comment:
Do not use plural form for entities and mind the errors:
`func Parse(filepath string) ([]Group, error)`